docs/CrossAccount.md

```diff
@@ -62,3 +63,109 @@ For security, auditing, and user-facing analytics, it is recommended that all
 S3 buckets in Quilt enable logging via CloudTrail. For cross-account buckets
 you must provide an existing trail to Quilt when you deploy the CloudFormation
 template, and you must add the buckets in question to CloudTrail.
+
+## Cross-account Simple Notification Service (Amazon SNS) topics
+
+To monitor events in an additional Amazon S3 bucket located in
+another account, *DataAccountTwo*, from the *ControlAccount*, you'll
+need to grant the *ControlAccount* access to *DataAccountTwo* bucket event
+notifications.
+
+This is done by adding a new IAM policy in the
+relevant *DataAccountTwo* Amazon S3 bucket that provides permissions
+to write to an existing SNS topic in the *ControlAccount*.
+
+1. Create a S3 bucket policy in the *DataAccountTwo* account that
+allows the *ControlAccount* access to all S3 events:
+
+<!--pytest.mark.skip-->
+```json
```

Comment:
This looks the same as the policy above, let's avoid duplication so we won't have to update it multiple places.

Reply:
Done, although I think it's now less readable and more error prone.